src/hotspot/share/runtime/javaCalls.cpp

```diff
@@ -347,9 +348,6 @@ Handle JavaCalls::construct_new_instance(InstanceKlass* klass, Symbol* construct
 void JavaCalls::call(JavaValue* result, const methodHandle& method, JavaCallArguments* args, TRAPS) {
   // Check if we need to wrap a potential OS exception handler around thread.
   // This is used for e.g. Win32 structured exception handlers.
-  assert(!UseWispMonitor || !is_init_completed() ||
```

Comment:
This will cause build error in debug mode and this line will be deleted in later patch(https://github.com/dragonwell-project/dragonwell11/commit/69d7997da583f2c478dc7029b80ca1b73d55fcb2). We shifted this change here.